test.js

```diff
@@ -113,3 +113,15 @@ test('bits and binary option', t => {
 	t.is(prettyBytes(1025, {bits: true, binary: true}), '1 kibit');
 	t.is(prettyBytes(1e6, {bits: true, binary: true}), '977 kibit');
 });
+
+test('fraction digits option', t => {
+	t.is(prettyBytes(1900, {maximumFractionDigits: 1}), '1.9 kB');
+	t.is(prettyBytes(1900, {minimumFractionDigits: 3}), '1.900 kB');
+	t.is(prettyBytes(1911, {maximumFractionDigits: 1}), '1.9 kB');
+	t.is(prettyBytes(1111, {maximumFractionDigits: 2}), '1.11 kB');
+	t.is(prettyBytes(1019, {maximumFractionDigits: 3}), '1.019 kB');
+	t.is(prettyBytes(1001, {maximumFractionDigits: 3}), '1.001 kB');
+	t.is(prettyBytes(4001, {maximumFractionDigits: 3, binary: true}), '3.907 kiB');
+	t.is(prettyBytes(18717, {maximumFractionDigits: 2, binary: true}), '18.28 kiB');
+	t.is(prettyBytes(18717, {maximumFractionDigits: 4, binary: true}), '18.2783 kiB');
```

Comment:
Needs some tests combining the two options.

Reply:
Done.